packages/react-components/react-alert/src/components/Alert/Alert.types.ts

```diff
@@ -1,15 +1,27 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { Button } from '@fluentui/react-button';
```

Comment:
_nit:_ We normally like to keep `import type` statements after regular `import` ones

```suggestion
import { Button } from '@fluentui/react-button';
import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
```